src/dotty/tools/dotc/typer/Checking.scala

```diff
@@ -156,16 +156,24 @@ object Checking {
         tp
     }
 
-    def apply(tp: Type) = tp match {
+    private def apply(tp: Type, cycleOK: Boolean, nestedCycleOK: Boolean): Type = {
```

Comment:
I picked the same names intentionally. I think there's also value in making clear they are the same.